pystiche_papers/johnson_alahi_li_2016/_data.py

```diff
@@ -30,6 +30,15 @@
 ]
 
 
+def license_info() -> str:
+    license_text = (
+        "The image is part of a repository that is published for personal and "
+        "research use only "
+        "(https://github.com/jcjohnson/fast-neural-style/blob/master/README.md#license)."
+    )
+    return license_text
```

Comment:
Since this functions takes no inputs and always returns the same output, can't we make it a constant?

```suggestion
LICENSE = (
    "The image is part of a repository that is published for personal and "
    "research use only "
    "(https://github.com/jcjohnson/fast-neural-style/blob/master/README.md#license)."
    )
```